NEWS.md

```diff
@@ -2,6 +2,7 @@
 
 * Automatically generating a usage section for an infix function containing "<-"
   no longer removes "<-" from the function name (#554).
+* Allowing empty lines in .Rbuildignore. Previously, empty lines caused all files being ignored.
```

Comment:
Can you please add a blank line before and add your github username. And mention issue #572 which this PR fixes